src/data/products.ts

```diff
@@ -0,0 +1,46 @@
+import { IProduct } from '../models';
+
+export const products: IProduct[] = [
+  {
+    id: 1,
+    name: 'Cheeseburger pizza',
+    img: 'cheeseburger-pizza.png',
+    category: 'all, meat',
```

Comment:
1. The category `all` looks redundant.
2. If you have a list of something, it will be much clearer to use an array